src/lib/journey/stages/webauthn.svelte

```diff
@@ -92,10 +105,11 @@
     </div>
   {/if}
 
+  {#if askForDeviceName}
   <div class="tw_text-center tw_w-full tw_py-4">
     <Spinner colorClass="tw_text-primary-light" layoutClasses="tw_h-28 tw_w-28" />
   </div>
-
+  {/if}
```

Comment:
I'm not sure I understand this condition. If `askForDeviceName` is `false`, we don't show a spinner? If we are to collect the device name, we do show a spinner. Am I missing something?